server/s3_error_codes.cc

```diff
@@ -34,6 +34,10 @@ int S3Error::get_http_status_code() { return details.get_http_status_code(); }
 void S3Error::set_auth_error_message(const std::string& errmsg) {
   auth_error_message = errmsg;
 }
+void S3Error::set_invalid_arg(const std::string& arg_name,
+                              const std::string& arg_val) {
+  invalid_args[std::move(arg_name)] = std::move(arg_val);
```

Comment:
std::move on a constant has no effect. Either get rid of the std::move calls, or change the args to values.

Reply:
removed std::move().